Changes to the Jenkins Integration README (#518)
jenkinsci/datadog-plugin: README.md

6. (optional) Enter your Trace Collection Port and select "Enable CI Visibility", optionally configuring your CI Instance name.
7. Save your configuration.

### Port 7 usage 

> The Datadog Agent sends a SYN packet to port 7 when new build nodes come online. The server is expected to respond with an RST packet (reset). The Agent uses this connection to test if it can reach the Jenkins server. An RST response (rather than no response) confirms that:

Author: ```suggestion
The Datadog Agent sends a SYN packet to port 7 when new build nodes come online to confirm connectivity. The Agent uses this connection to test if it can reach the Jenkins server. The server is expected to respond with an RST packet (reset). An RST response (rather than no response) confirms that:
```

> The Datadog Agent sends a SYN packet to port 7 when new build nodes come online to confirm connectivity. The Agent uses this connection to test if it can reach the Jenkins server. The server is expected to respond with an RST (reset) packet. An RST response (rather than no response) confirms that:
> 
> 1. The network path exists
> 2. The server is reachable
> 3. The server is actively rejecting the connection (which is the desired outcome)

Author: ```suggestion
When a Jenkins Agent node connects to the controller, the plugin on the controller sends a corresponding event to Datadog. During event creation, the plugin calls Jenkins core APIs to retrieve agent metadata (such as hostname and labels). These API calls internally perform several checks, including a reachability test using java.net.InetAddress#isReachable. This call sends a TCP SYN packet from the controller to port 7 on the Agent node. The server on the Agent node responds with a RST packet (Reset).

This is a common technique used to verify network connectivity between services. The fact that it gets a RST response (rather than no response) confirms that:

1. The network path exists
2. The server is reachable
3. The server is actively rejecting the connection (which is what we want)
```

6. (optional) Enter your Trace Collection Port and select "Enable CI Visibility", optionally configuring your CI Instance name.
7. Save your configuration.

### Port 7 usage 

> When a Jenkins Agent node connects to the controller, the plugin on the controller sends a corresponding event to Datadog. During event creation, the plugin calls Jenkins core APIs to retrieve agent metadata (such as hostname and labels). These API calls internally perform several checks, including a reachability test using java.net.InetAddress#isReachable. This call sends a TCP SYN packet from the controller to port 7 on the Agent node. The server on the Agent node responds with a RST packet (Reset).

Author: ```suggestion
When a Jenkins Agent node connects to the controller, the plugin on the controller sends a corresponding event to Datadog. During event creation, the plugin calls Jenkins core APIs to retrieve Agent metadata (such as hostname and labels). These API calls internally perform several checks, including a reachability test using java.net.InetAddress#isReachable. This call sends a TCP SYN packet from the controller to port 7 on the Agent node. The server on the Agent node responds with a RST packet (Reset).
```

Modified form agent to Agent
